default/AI/AIstate.py

```diff
@@ -662,6 +661,14 @@ def rate_fleet(self, fleet_id, enemy_stats=None, ship_info=None):
                 shields = ship.currentMeterValue(fo.meterType.shield)
                 stats['structure'] = structure
                 stats['shields'] = shields
+                stats['attack'] = 0
+                stats['attacks'] = {}
+                design = fo.getShipDesign(design_id)
+                for partname in design.parts:
```

Comment:
> Did the bug occur in multiplayer?

Actually, even more specific: I would assume the bug happened with you as moderator?